src/main/webapp/script.js

```diff
@@ -24,7 +24,8 @@ google.setOnLoadCallback(getChart)
 async function getYouTubeComments() { 
   const urlInput = document.getElementById('link-input');
   const url = cleanseUrl(urlInput.value);
-  const response = await fetch("/YouTubeComments?url="+url);
+  const numComments = document.getElementById('numComments-input').value;
```

Comment:
change "numComments-input" to a static final?

Reply:
I don't think that's necessary since it's only referenced once